packages/components/src/globals/scss/_tooltip.scss

```diff
@@ -8,7 +8,23 @@
 @import 'layer';
 
 // Tooltip
-// Definition and Icon CSS only tooltip
+// Forces visible state for definition and icon only tooltip
+/// @access public
+/// @group tooltip
+@mixin tooltip--visible() {
```

Comment:
Is this mixin meant to be public/used elsewhere? Seems unnecessary.

Reply:
I'm ok with making it private, the idea was to allow the "tooltip visible" rules to be applied to the tooltip trigger wrapper when the tooltip trigger itself cannot display the tooltip properly (e.g. when the trigger is an input field/form control)